### PR TITLE
Use different messages on lambda errors

### PR DESCRIPTION
**Pull request by elsteelbrain**

### CODE REVIEW

**Comment by elsteelbrain** [comment]
The syntax for `case` statement is correct. There must be a problem with the new `if` statement. I'll take a look again and update.

**Comment by elsteelbrain** [comment]
@alexdebrie The problem was that the previous implementation only changed error messages for HTTP events and not the invoke ones. I've updated the logic, so that, both cases are covered.

**Comment by alexdebrie** [comment]
@elsteelbrain This looks good by me now. Can you update the failing tests? Will wait on @mthenw for final approval.

File: router/router.go [hunk 1]
```diff
@@ -318,6 +317,29 @@ func (router *Router) handleInvokeEvent(space string, functionID function.ID, pa
 	}
 }
 
+func determineErrorMessage(err error) string {
```
Comment (mthenw) [inline router/router.go]: This function should not be defined in router package. It should be defined in function package. This is where providers specific logic should live.

**Pull request merged**